components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs

```diff
@@ -266,11 +283,13 @@ public class DefaultSessionManager : ISessionManager {
                 IEnumerable<EndpointDescription> filteredEndpoints = endpoints.ToArray();
 
```

Comment:
should filter here if endpoint security policy is in list of supportedpolicies (in Opc.Ua.SecurityPolicies) or unsupported policy can be seletced

Reply:
Don't you have a branch with that particular change?  Can we pull it from there?

Reply:
Put on backlog and will fix outside of this PR.